score/etl/base.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from config import settings
+from utils import unzip_file_from_url
+
+
+class Etl:
```

Comment:
nitty nit: I'm really averse to acronyms for a few different reasons, haha. How about `ExtractTransformLoad`?